src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -134,6 +134,28 @@ private void authorizeResourceAdmin(final Resource resource) throws AccessDenied
         }
     }
 
+    private void authorizeResourceView(final Resource resource) throws AccessDeniedException {
+        try {
+            if (!adminService.isAdmin(AuthorizationService.Operation.ADMIN)) {
+                if (!authorizationService.isAuthorized(AuthorizationService.Operation.VIEW, resource)) {
+                    throw new AccessDeniedException(AuthorizationService.Operation.VIEW, resource);
+                }
+            }
+        } catch (final PluginException e) {
+            throw new ServiceTemporarilyUnavailableException("Error calling authorization plugin", e);
+        }
+    }
+
+    public void authorizeEventTypeView(final EventType eventType)
+            throws AccessDeniedException, ServiceTemporarilyUnavailableException {
+        authorizeResourceView(eventType.asResource());
+    }
+
+    public void authorizeSubscriptionView(final Subscription subscription) throws
```

Comment:
it's better to move `throws` to a next line, like you did for a method above.

Reply:
Fixed this.